### PR TITLE
PR Annotations removed

### PR DESCRIPTION
**Pull request by jflo**
## PR description

Removes annotation of failed tests onto pr, since we have to run them before review anyway.
Updates nightly release with a single artifact.
Updates PR template order

### Thanks for sending a pull request! Have you done the following?

- [ ] Checked out our [contribution guidelines](https://github.com/hyperledger/besu/blob/main/CONTRIBUTING.md)?
- [ ] Considered documentation and added the `doc-change-required` label to this PR [if updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).
- [ ] Considered the changelog and included an [update if required](https://wiki.hyperledger.org/display/BESU/Changelog).
- [ ] For database changes (e.g. KeyValueSegmentIdentifier) considered compatibility and performed forwards and backwards compatibility tests

### Most advanced CI tests are deferred until PR approval, but you could:

- [ ] locally run all unit tests via: `./gradlew build`
- [ ] locally run all acceptance tests via: `./gradlew acceptanceTest`
- [ ] locally run all integration tests via: `./gradlew integrationTest`
- [ ] locally run all reference tests via: `./gradlew ethereum:referenceTests:referenceTests`

### CODE REVIEW

File: .github/workflows/pre-review.yml [hunk 1]
```diff
@@ -108,12 +124,11 @@ jobs:
       - name: run unit tests
         id: unitTest
         run: ./gradlew $GRADLEW_UNIT_TEST_ARGS
-      - name: Publish Test Report
-        uses: mikepenz/action-junit-report@5f47764eec0e1c1f19f40c8e60a5ba47e47015c5
-        if: success() || failure() # always run even if the build step fails
```
Comment (macfarla) [inline .github/workflows/pre-review.yml:113]: so Publish Test Report doesn't happen any more?

File: .github/workflows/develop.yml [hunk 2]
```diff
@@ -28,24 +28,14 @@ jobs:
...
-          path: 'build/distributions/besu*.tar.gz'
-          name: besu-develop.tar.gz
-          compression-level: 0
-      - name: hashes
-        id: hashes
+      - name: rename and hash files
         run: |
           cd build/distributions
+          mv besu*.zip besu-develop.zip
+          mv besu*.tar.gz besu-develop.zip
```
Comment (macfarla) [inline .github/workflows/develop.yml:35]:
```suggestion
          mv besu*.tar.gz besu-develop.tar.gz
```

**Review by macfarla — COMMENTED** [review]
one typo 
one question about publish test report

File: .github/workflows/pre-review.yml (hunk 1 above)
Comment (jflo) [reply .github/workflows/pre-review.yml:113]: correct.  annotating the PR requires write access to the repository that the PR lives in, which is the targeted fork. If we want to allow this, we would need to switch the event type back to `pull_request_target` which would then run in the context of the targeted fork, using the version of the workflow defined there.

**Review by garyschulte — APPROVED** [review]
🚢

**Pull request merged**